src/revlanguage/main.cpp

```diff
@@ -28,14 +28,38 @@ using po::variables_map;
 
 std::string usage()
 {
-    return "Usage: rb [OPTIONS]\n       rb [OPTIONS] <file1> [<file2> ...]\n";
-    // Other usages not mentioned
+    std::string usage =
+        "Usage: rb [OPTIONS]\n"
+        "       rb [OPTIONS] <file1> [<file2> ...] [--args <arg1> [<arg2> ...]]";
+
+    return usage;
+}
+
+
+std::string usage_examples()
+{
+    std::string usage_examples =
+        "Usage examples:\n"
+        "   1. rb\n"
+        "   2. rb --args 1 2\n"
```

Comment:
Supplying arguments without a script seems odd.

In general, some of these examples seem like they are intended more to explain the parser than to be used as actual examples.

Reply:
Yeah, it's not likely to be a common use case – I can maybe see it being used in classroom / workshop settings precisely to demonstrate how the `--args` flag works. The `?args` help page relies on this usage for the same reason.